pkg/cmd/cli/common.go

```diff
@@ -18,11 +18,65 @@ package cli
 
 import (
 	"bufio"
+	"errors"
 	"fmt"
 	"os"
 	"strings"
+
+	"github.com/heptio/ark/pkg/client"
+	"github.com/heptio/ark/pkg/cmd/util/flag"
+	clientset "github.com/heptio/ark/pkg/generated/clientset/versioned"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
+// DeleteOptions contains parameters used for deleting a restore.
+type DeleteOptions struct {
+	Names     []string
+	All       bool
```

Comment:
`All` and any other element that is not used outside this package can be unexported.